packages/ui/src/proposals/components/ProposalDetails/ProposalDetails.tsx

```diff
@@ -51,9 +53,10 @@ const renderTypeMapper: Partial<Record<RenderType, ProposalDetailContent>> = {
   Percentage: Percentage,
   Hash: Hash,
   DestinationsPreview: DestinationsPreview,
+  BlockDurationStatistics: BlockDurationStatistics,
```

Comment:
It shows the right thing but I think this should be the only way we show a number of blocks. So instead of adding:
```ts
BlockDurationStatistics: BlockDurationStatistics,
```
Please replace `NumberOfBlocks` by:
```ts
NumberOfBlocks: BlockDurationStatistics,
```
and remove `NumberOfBlocks.ts`